app/controllers/api/account.php

```diff
@@ -3648,12 +3648,119 @@ function sendSessionAlert(Locale $locale, Document $user, Document $project, Doc
+
+                $body
+                    ->setParam('{{buttonText}}', $buttonText)
+                    ->setParam('{{redirect}}', $redirect)
+                    ->setParam('{{body}}', $bodyText, escapeHtml: false)
+                    ->setParam('{{project}}', $project->getAttribute('name'))
+                    ->setParam('{{user}}', $user->getAttribute('name'));
+
+                $message->setParam('{{body}}', $body->render(), escapeHtml: false);
+            } else {
+                $message = Template::fromFile(__DIR__ . '/../../config/locale/templates/email-mfa-altered-client.tpl');
+
+                $message->setParam('{{body}}', $locale->getText("emails.{$type}.body"), escapeHtml: false);
+            }
+
+            $message
+                ->setParam('{{hello}}', $locale->getText("emails.{$type}.hello"))
+                ->setParam('{{footer}}', $locale->getText("emails.{$type}.footer"))
+                ->setParam('{{thanks}}', $locale->getText("emails.{$type}.thanks"))
+                ->setParam('{{signature}}', $locale->getText("emails.{$type}.signature"));
+
+            $body = $message->render();
+
+            $smtp = $project->getAttribute('smtp', []);
+            $smtpEnabled = $smtp['enabled'] ?? false;
+
+            $senderEmail = System::getEnv('_APP_SYSTEM_EMAIL_ADDRESS', APP_EMAIL_TEAM);
+            $senderName = System::getEnv('_APP_SYSTEM_EMAIL_NAME', APP_NAME . ' Server');
+            $replyTo = "";
+
+            if ($smtpEnabled) {
+                if (!empty($smtp['senderEmail'])) {
+                    $senderEmail = $smtp['senderEmail'];
+                }
+                if (!empty($smtp['senderName'])) {
+                    $senderName = $smtp['senderName'];
+                }
+                if (!empty($smtp['replyTo'])) {
+                    $replyTo = $smtp['replyTo'];
+                }
+
+                $queueForMails
+                    ->setSmtpHost($smtp['host'] ?? '')
+                    ->setSmtpPort($smtp['port'] ?? '')
+                    ->setSmtpUsername($smtp['username'] ?? '')
+                    ->setSmtpPassword($smtp['password'] ?? '')
+                    ->setSmtpSecure($smtp['secure'] ?? '');
+
+                if (!empty($customTemplate)) {
+                    if (!empty($customTemplate['senderEmail'])) {
+                        $senderEmail = $customTemplate['senderEmail'];
+                    }
+                    if (!empty($customTemplate['senderName'])) {
+                        $senderName = $customTemplate['senderName'];
+                    }
+                    if (!empty($customTemplate['replyTo'])) {
+                        $replyTo = $customTemplate['replyTo'];
+                    }
+
+                    $body = $customTemplate['message'] ?? '';
+                    $subject = $customTemplate['subject'] ?? $subject;
+                }
```

Comment:
When a developer tries to customize the template for their project, the Console makes a `GET /v1/projects/acme-app/templates/email/mfaEnabled/en` API call to fetch the default template. It looks like this is returning the Console template and not the Project template:

```json
{
    "type": "mfaEnabled",
    "locale": "en",
    "message": "<p>Hello {{user}},</p>\n<p>Two-factor authentication has been successfully enabled for your {{project}} account.</p>\n<p><a href=\"{{redirect}}\" target=\"_blank\">{{redirect}}</a></p>\n<p>If you didn’t enable two-factor authentication, please contact us immediately.</p>\n<p style=\"margin-bottom: 32px\">\n    Thanks,\n    <br/>\n    {{project}} team\n</p>",
    "senderName": "",
    "senderEmail": "",
    "replyTo": "",
    "subject": "Two-factor authentication is enabled for your {{project}} account"
}
```

Also, getting a non-english locale like `GET /v1/projects/acme-app/templates/email/mfaEnabled/es` returns a template some of the variables:

```json
{
    "type": "mfaEnabled",
    "locale": "es",
    "message": "<p>{{emails.mfaEnabled.hello}},</p>\n<p>{{emails.mfaEnabled.body}}</p>\n<p><a href=\"{{redirect}}\" target=\"_blank\">{{redirect}}</a></p>\n<p>{{emails.mfaEnabled.footer}}</p>\n<p style=\"margin-bottom: 32px\">\n    {{emails.mfaEnabled.thanks}},\n    <br/>\n    {{emails.mfaEnabled.signature}}\n</p>",
    "senderName": "",
    "senderEmail": "",
    "replyTo": "",
    "subject": "{{emails.mfaEnabled.subject}}"
}
```

I don't think we need to work on this now, but it would be good to create an issue or something to track the work.